jmbitcoin/jmbitcoin/secp256k1_transaction.py

```diff
@@ -460,7 +460,7 @@ def script_to_address(script, vbyte=b'\x00', witver=0):
         return bech32addr_encode(hrp=hrp, witver=witver,
         witprog=struct.unpack('{}B'.format(len(script[2:])).encode(
             'ascii'), script[2:]))
-    if is_p2pkh_script(script):
+    elif is_p2pkh_script(script):
```

Comment:
Not needed, I don't think it aids readability, since the previous if unconditionally returns.

Reply:
OK, I will undo